about.html

```diff
@@ -18,20 +18,47 @@
     <header>
         <div class="heading-background">
             <nav id="nav-section">
-                <i class="fa-solid fa-bars" id="menu"></i>
+                <a href="./index.html"><i class="fa-solid fa-bars" id="menu"></i></a>
```

Comment:
- [ ] [Optional] It would be better to display a menu when clicking on the hamburger button instead of directing the user to the home page.